seatunnel-e2e/seatunnel-connector-v2-e2e/connector-hive-e2e/src/test/java/org/apache/seatunnel/e2e/connector/hive/HiveIT.java

```diff
@@ -153,16 +232,22 @@ public void testFakeSinkHiveOnHDFS(TestContainer container) throws Exception {
     }
```

Comment:
I found we only have test case of write to hive, could you add a test case read from hive?